### DISCUSSION TITLE
How to properly setup TLS with webrtc docker ome

### DISCUSSION THREAD

**Discussion by devintly** [OvenMediaLabs/OvenMediaEngine#1363] [Help]
I have problem when start streaming from obs by webrtc whip:
`[2023-09-01 14:30:16.984] E [SPRtcSig-t3334:11] OpenSSL | tls.cpp:208  | An error occurred while accept SSL connection: [OpenSSL] error:0A0000C1:SSL routines::no shared cipher (167772353)`

I get my certificate throught Nginx Proxy Manager

OME Running in Docker on UBUNTU VPS

OBS string: https://ip:3334/app/yalsquad227756?direction=whip

My Server.xml:
[ServerCopy.txt](https://github.com/AirenSoft/OvenMediaEngine/files/12499386/ServerCopy.txt)

**Comment by devintly** [comment]
Now tried zerossl certificate, now error is `[2023-09-01 18:31:59.481] E [SPRtcSig-t3334:12] OpenSSL | tls.cpp:208  | An error occurred while accept SSL connection: [OpenSSL] error:00000000:lib(0)::reason(0) (0)`

What should i do?
Thanks in advance!

**Reply by getroot** [reply]
Isn't it because it's connected via IP? Certificates mostly work with domains.
